internal/p2p/metrics.go

```diff
@@ -28,6 +28,9 @@ var (
 type Metrics struct {
 	// Number of peers.
 	Peers metrics.Gauge
```

Comment:
Do we want to call this `PeersConnected`

Reply:
https://github.com/tendermint/tendermint/blob/p2p-dialer-store-change/internal/p2p/router.go#L757

Reply:
Yes, I was suggesting we rename `Peers` to `PeersConnected` to make it less ambiguous

Reply:
done!